webhook/admission.go

```diff
@@ -101,9 +101,12 @@ func admissionHandler(rootLogger *zap.SugaredLogger, stats StatsReporter, c Admi
 			patchType = string(*reviewResponse.PatchType)
 		}
 
-		logger.Infof("AdmissionReview for %#v: %s/%s response={ UID: %#v, Allowed: %t, Status: %#v, Patch: %s, PatchType: %s, AuditAnnotations: %#v}",
+		logger.Infof("AdmissionReview for %#v: %s/%s response={ UID: %#v, Allowed: %t, Status: %#v, PatchType: %s, AuditAnnotations: %#v}",
```

Comment:
I wonder if we want to store a few of these fields (status, allowed, patchType) using `.With` for structured logging, rather than burying them in a string.

Reply:
Done. Also remove some admission request info from the message since they have already stored in logging context.